SHIR/build.ps1

```diff
@@ -39,5 +70,6 @@ function SetupEnv() {
 }
 
 Install-SHIR
+Install-MSFT-JDK
```

Comment:
Maybe add a build-time variable check to skip JDK installation.

Reply:
I will gladly do this if Azure says I should for this pull request to be merged.

Reply:
> Maybe add a build-time variable check to skip JDK installation.

agree